lib/oli_web/live/workspaces/course_author/curriculum/entry.ex

```diff
@@ -52,7 +53,9 @@ defmodule OliWeb.Workspaces.CourseAuthor.Curriculum.Entry do
             <span class="ml-1 mr-1 entry-title"><%= @child.title %></span>
             <.link
               class="entry-title mx-3"
-              href={~p"/workspaces/course_author/#{@project.slug}/curriculum/#{@child.slug}/edit"}
+              href={
```

Comment:
This might be replaced with a `navigate` attribute, but we need to inject all the JS code to render the React components. Due to time constraints, I'll leave this as it is.